Automattic/jetpack
# Shared Extension Utilities: Initial setup of the package

#### Changes proposed in this Pull Request:

This is the skeleton of the new shared-extension-utils package that will
house the code that currently resides in the `extensions/shared` folder
of the Jetpack plugin.

#### Jetpack product discussion
p9dueE-4cw-p2#comment-7082

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
There probably isn't much to test currently. Maybe we just need to check that it's setup correctly and that `jetpack install --all` works.
